controller/RestTestCenter.php

```diff
@@ -33,6 +37,104 @@ class RestTestCenter extends AbstractRestController
     const PARAMETER_TEST_CENTER_CLASS = 'class';
     const PARAMETER_TEST_CENTER_LABEL = 'label';
 
+    /**
+     * @var array
+     */
+    protected $parametersMap = [
+        self::PARAMETER_TEST_CENTER_LABEL => OntologyRdfs::RDFS_LABEL
+    ];
+
+    /**
+     * @OA\Put(
+     *     path="/taoOffline/api/testCenter",
```

Comment:
you need to change this for this  extension (taoTestCenter instead of taoOffline)